### ISSUE TITLE
Documenting a method will make the cursor jump… somewhere else.

### ISSUE THREAD

**Issue by Elarcis** [joelday/vscode-docthis#89]
When I do Ctrl+Alt+D Ctrl+Alt+D to comment a method while my cursor is situated **any amount of whitespace before** my documented thing, my cursor is moved somewhere else in the file.

It seems the place is determined by the height of the generated documentation. Would be great if it moved the cursor at the place of the documentation's description instead.

![demo](http://g.recordit.co/oi9HMrNsb2.gif)

Tested on VSCode 1.9, with and without CodeLens references enabled.

**Comment by joelday** [comment]
Unable to reproduce this. Can you try again and reopen?

**Comment by TimMensch** [comment]
I get this *every single time*. Except when DocumentThis throws an error, which is what happened the first time I tried it to reproduce the bug. Turns out if you use DocumentThis on a "new" file that isn't saved, it just crashes.

But back to the bug at hand: If you create [this file](http://pastebin.com/7Ftw2gpF) and put the cursor on the first character of line 7 (with the `f()` definition), then click `Ctrl-Alt-D Ctrl-Alt-D`, it throws the cursor up like in the image above.

Also behaves poorly if you put your cursor just after the last `}` in `f()`; it documents `g()` but throws the cursor somewhere above `f()`.

The key seems to be: The cursor is *before* the thing you're trying to document.
